### ISSUE TITLE
Image generation fails with nix-env being aborted

### ISSUE THREAD

**Issue by software-artificer** [filakhtov/nixos-conjurer#1]
Running `nixos-conjurer config.yaml` currently results in the following failure:
```
Configure and update Nix channels...
... ERROR: failed to update Nix channels: the `nix-channel` command returned non-zero status:
Output { status: ExitStatus(unix_wait_status(256)), stdout: "", stderr: "unpacking channels...\nAssertion failed: settings.buiacquire: 25)\nerror: program '/usr/bin/nix-env' failed due to signal 6 (Aborted)\n" }
```

**Comment by software-artificer** [comment]
Upon investigation, I have discovered that [this commit](https://github.com/NixOS/nix/commit/8c4cce553c16438f0ccbbaea6d77f2bd64306dfe#diff-a737fa50e88bc9fbc319594c5f9eef20b1d42546a2dba204d622c6b46f71ebfeR215) released as part of 2.12.0 breaks an ability to run `nix` with empty `build-users-group`, which `nixos-conjurer` relies on.

It was subsequently discovered and fixed in 2.13.0 in [this commit](https://github.com/NixOS/nix/commit/1f3c0a3c1dde7e0348c085e0fbb60729cb067d86).

However, Alpine currently packaging 2.12.0 and thus is affected by this problem.

**Comment by software-artificer** [comment]
The [Nix package](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/nix) is now at 2.13.2 in Alpine and the issue is fixed.
